benchmark_runner/common/oc/oc.py

```diff
@@ -23,6 +23,42 @@ def __init__(self, kubeadmin_password: str = ''):
         super().__init__()
         self.__kubeadmin_password = kubeadmin_password
 
+    def get_ocp_server_version(self):
+        """
+        This method return ocp server version
+        :return:
+        """
+        ocp_version_data = self.run('oc version')
```

Comment:
At some point all of this should use the `openshift` package; we'll want to pay down that technical debt.